MiniAn/minian_main.py

```diff
@@ -467,7 +470,113 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(AC, A, minian_parameters):
+    perform_cross_reg = minian_parameters.params_cross_reg["crossReg"]
+    if not perform_cross_reg:
+        return A
+    param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
+    ref_filename = minian_parameters.params_cross_reg["fname"]
+    ref_images   = minian_parameters.params_cross_reg["h5path_images"]
+    ref_ROIs     = minian_parameters.params_cross_reg["h5path_roi"]
+
+    # Load base/reference file to xarray
+    ref, file_ref_ =  load_doric_to_xarray(ref_filename, ref_images, np.float64, None, "subset", None)
```

Comment:
Make sure to close the reference file

Reply:
`file_` has the `_`, because `file` is reserved by python, `file_ref` isn't, so no need in extra `_`